gooddata-sdk/gooddata_sdk/catalog/workspace/model/workspace.py

```diff
@@ -0,0 +1,13 @@
+# (C) 2022 GoodData Corporation
+from __future__ import annotations
+
+from gooddata_sdk.catalog.entity import CatalogEntity
+
+
+class CatalogWorkspace(CatalogEntity):
+    @property
+    def name(self) -> str:
+        return self._e["name"]
+
+    def __repr__(self) -> str:
```

Comment:
This is not needed. Check out parent – CatologEntity implementation of method __repr__.